### DISCUSSION TITLE
FR: "Has descendants" condition

### DISCUSSION THREAD

**Discussion by mmikkel** [craftcms/cms#15276] [Ideas] [answered]
It'd be awesome if the condition builder included a "Has descendants" boolean condition.

(The "Level" condition can sometimes be used, but not necessarily in cases where there isn't a max level set for the structure).

**Comment by brandonkelly** [comment, marked as answer]
Easy enough! Added for Craft 4.11 and 5.3 (315ccfaf3d73ae611b159f14814ca38bbf47c813).

**Reply by mmikkel** [reply]
Hooray! This is going to make my life a lot easier on a current project. Thanks Brandon!

**Reply by brandonkelly** [reply]
5.3.0-beta.1 has been tagged 🎉
